incubator/hnc/pkg/reconcilers/hierarchical_namespace_test.go

```diff
@@ -25,46 +26,124 @@ var _ = Describe("Hierarchy", func() {
 
 		fooName = createNS(ctx, "foo")
 		barName = createNSName("bar")
+	})
 
-		// Create "bar" hns in "foo" namespace
+	It("should set the hns.status.state to Ok if the subnamespace is created succussfully", func() {
```

Comment:
This test now does a lot of things, maybe generalize the description?

Reply:
Do you mean in a comment above this or in the "should..." string?